detox/src/realms/DetoxPrimaryContext.test.js

```diff
@@ -1,62 +1,73 @@
-// @ts-nocheck
 const _ = require('lodash');
 
 const {
+  backupProcessEnv,
   latestInstanceOf,
   lastCallTo,
-  throwErrorImpl,
-  withSuspendingMock,
-  uuidRegexp,
-  tempFileRegexp,
-} = global;
-
-const workerId = 19;
+} = require('../../__tests__/helpers');
 
 describe('DetoxPrimaryContext', () => {
+  //#region Fixtures and constants
+  const UUID_REGEXP_STR = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}';
+  const UUID_REGEXP = new RegExp(`^${UUID_REGEXP_STR}$`);
+  const TEMP_FILE_REGEXP = new RegExp(`.*${UUID_REGEXP_STR}.detox.json$`);
+  const WORKER_ID = 19;
+  const DETOX_CONFIG_BASE = Object.freeze({
+    behavior: {
+      init: {
+        keepLockFile: false,
+      }
+    },
+    device: {
+      type: '',
+    },
+    logger: {},
+    session: {
+      autoStart: false,
+    },
+  });
+  const FIRST_ARGUMENT = 0;
+  //#endregion
+
   let detoxConfigDriver;
 
+  //#region Mocks
+  /** @type {jest.Mocked<import('fs-extra')>} */
```

Comment:
I fixed the types and extracted all mocked classes and instances to the region with those `let`, `let` ... `let` variables.

Reply:
OK but all of those `#region` comments are beginning to mess up our code (e.g. if you have Webstorm as an IDE, or on Github). I'd rather hold that back a bit.